device-types/Ciena/6500-R4.yaml

```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Ciena
+model: 6500 R4
+slug: ciena-6500-r4
+part_number: NTK803FA
+u_height: 4
+is_full_depth: true
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it